aws/dns.tf

```diff
@@ -88,3 +88,11 @@ resource "aws_route53_record" "web_auth" {
   type    = "A"
   records = concat(aws_eip.cluster[*].public_ip)
 }
+
+resource "aws_route53_record" "crl" {
+  zone_id = aws_route53_zone.cluster.id
+  name    = "crl.${aws_route53_zone.cluster.name}"
+  ttl     = 300
+  type    = "CNAME"
+  records = ["crl.${aws_route53_zone.cluster.name}.s3.${var.region}.amazonaws.com."]
```

Comment:
Fine as is, but it may be good to clean this up in case someone changes the convention on the bucket

records = ["crl.${aws_s3_bucket.veto_crls.bucket_regional_domain_name}"]

Reply:
There seems to be an open issue saying that this doesn't work as expected https://github.com/hashicorp/terraform-provider-aws/issues/15102

I guess is something that we would need to check.